feg/radius/src/modules/lbserve/lb_serve_test.go

```diff
@@ -206,21 +208,20 @@ func spawnFailingRadiusServer() (server *radius.PacketServer, port int) {
 		),
 		SecretSource: radius.StaticSecretSource(secret),
 		Addr:         fmt.Sprintf(":%d", port),
-		Ready:        make(chan bool, 1),
 	}
 	fmt.Print("Starting server... ")
 	go func() {
 		_ = server.ListenAndServe()
 	}()
-	<-server.Ready // Wait for server to get ready
+	time.Sleep(10 * time.Millisecond)
 	fmt.Println("Server listening")
 	return
 }
 
 func createRadiusRequest() *radius.Request {
 	packet := radius.New(radius.CodeAccessRequest, []byte{0x01, 0x02, 0x03, 0x4, 0x05, 0x06})
-	packet.Attributes[rfc2865.CallingStationID_Type] = []radius.Attribute{radius.Attribute("called")}
-	packet.Attributes[rfc2865.CalledStationID_Type] = []radius.Attribute{radius.Attribute("calling")}
+	packet.Add(rfc2865.CallingStationID_Type, radius.Attribute("calling"))
+	packet.Add(rfc2865.CalledStationID_Type, radius.Attribute("called"))
```

Comment:
We are turning `calling` and `called` around here. Are we sure this is correct? At the least this should be mentioned in the PR summary.

Reply:
I changed it back to how it was originally. Everywhere else these attributes are switched, but in the end it doesn't matter at all what's written here, as long as the attributes are not `nil`, as that's the only thing that's asserted in the tests..